Documentation/Legacy/FrameworkOverview.md

```diff
@@ -8,213 +8,196 @@ learning about new modules and finding more specific documentation.
-serial execution queue to perform work or deliver results upon.
-
-[Signals](#signals) can be ordered to deliver events a a specific scheduler or
-[SignalProducers](#signal-producers) can be ordered to start their work on 
-a specific scheduler.
+A **scheduler**, represented by the [RACScheduler][] class, is a serial
+execution queue for [signals](#signals) to perform work or deliver their results upon.
 
 Schedulers are similar to Grand Central Dispatch queues, but schedulers support
 cancellation (via [disposables](#disposables)), and always execute serially.
-With the exception of the [ImmediateScheduler][Scheduler], schedulers do not
+With the exception of the [+immediateScheduler][RACScheduler], schedulers do not
 offer synchronous execution. This helps avoid deadlocks, and encourages the use
-of [signal operators][BasicOperators] instead of blocking work.
+of [signal operators][RACSignal+Operations] instead of blocking work.
 
-[Schedulers][Scheduler] are also somewhat similar to `NSOperationQueue`, but schedulers
+[RACScheduler][] is also somewhat similar to `NSOperationQueue`, but schedulers
 do not allow tasks to be reordered or depend on one another.
 
+## Value types
+
+RAC offers a few miscellaneous classes for conveniently representing values in
+a [stream](#streams):
+
+ * **[RACTuple][]** is a small, constant-sized collection that can contain
+   `nil` (represented by `RACTupleNil`). It is generally used to represent
+   the combined values of multiple streams.
+ * **[RACUnit][]** is a singleton "empty" value. It is used as a value in
+   a stream for those times when more meaningful data doesn't exist.
+ * **[RACEvent][]** represents any [signal event](#signals) as a single value.
+   It is primarily used by the `-materialize` method of
+   [RACSignal][RACSignal+Operations].
 
 [Design Guidelines]: DesignGuidelines.md
+[Haskell]: http://www.haskell.org
+[lazy-seq]: http://clojure.github.com/clojure/clojure.core-api.html#clojure.core/lazy-seq
+[List]: https://downloads.haskell.org/~ghc/latest/docs/html/libraries/base-4.7.0.2/Data-List.html
 [Memory Management]: MemoryManagement.md
-[BasicOperators]: BasicOperators.md
-[README]: ../README.md
-[Signal]: ../ReactiveCocoa/Swift/Signal.swift
-[SignalProducer]: ../ReactiveCocoa/Swift/SignalProducer.swift
-[Action]: ../ReactiveCocoa/Swift/Action.swift
-[CocoaAction]: ../ReactiveCocoa/Swift/Action.swift
-[Disposable]: ../ReactiveCocoa/Swift/Disposable.swift
-[Scheduler]: ../ReactiveCocoa/Swift/Scheduler.swift
-[Property]: ../ReactiveCocoa/Swift/Property.swift
-[Event]: ../ReactiveCocoa/Swift/Event.swift
-[SinkOf]: http://swiftdoc.org/type/SinkOf/
+[monads]: http://en.wikipedia.org/wiki/Monad_(functional_programming)
+[Monoid]: http://downloads.haskell.org/~ghc/latest/docs/html/libraries/base-4.7.0.2/Data-Monoid.html
+[MonadZip]: http://downloads.haskell.org/~ghc/latest/docs/html/libraries/base-4.7.0.2/Control-Monad-Zip.html
```

Comment:
This link does not work anymore, I think this may be the current link:
http://downloads.haskell.org/~ghc/latest/docs/html/libraries/Control-Monad-Zip.html